### ISSUE TITLE
Anki Windows desktop 2.1.28 cannot sync 

### ISSUE THREAD

**Issue by dobefore** [tsudoko/anki-sync-server#77]
It shows,`username or password is incorrect,please try again`,Yet,anki sync server printed nothing.thanks.

**Comment by okay456okay** [comment]
+1

**Comment by controlledentropy** [comment]
> +1

+1，and ankidroid must ask for https config for sake of google request from android 10

**Comment by dobefore** [comment]
> ankidroid must ask for https config for sake of google request from android 10

for local https request,mkcert(a tool for getting rootCA) and revision of sync_app.py can work. 
for public   IP,let's encripted and reverse proxy can work

**Comment by controlledentropy** [comment]
> > ankidroid must ask for https config for sake of google request from android 10
> 
> for local https request,mkcert(a tool for getting rootCA) and revision of sync_app.py can work.
> for public IP,let's encripted and reverse proxy can work

thanks for your reply : )
my pie's repository is not include mkcert so I generated certs by openssl, anyway, android can not install .pem so i changed to .cert but cant sync(I'll figure out after exam）

a new issue is about windows ankiweb 2.1.29. i tried to create a new profile to sync, "incorrect username or password" dialog disappeared, and sync process seems normal. but then I found the data synced from server is just a part of the whole.

then I even deployed a completely new windows10 2004 on an old laptop then tried ankiweb 2.1.15 as well as 2.1.29. finally I ensure that the problem. 2.1.29 can't download whole data(for my occasion there remains 2 big deck disappeared but successfully synced to 2.1.15). I guess maybe the problem is originated from sync mechanism change(data first, media second).

but for not enough spare time, I just abandon https and further research. hope issues will be solved by other buddies XD
